documentation/synthetic_tokens/tutorials/creating_from_sponsor_cli.md

```diff
@@ -0,0 +1,205 @@
+# UMA Sponsor CLI Tool
+
+The Sponsor CLI tool lets you interact with a synthetic token contract.
+In this tutorial, you’ll be deploying a synthetic token contract to a local testnet and interacting with it via the Sponsor CLI tool.
+
+It will let you be the “Token Sponsor,” which lets you:
+
+- Deposit collateral (ETH) and borrow synthetic tokens
+- Manage your position’s collateral (deposit/withdraw)
+- Redeem the borrowed synthetic tokens and get collateral back
+- Transfer your position to another Ethereum address
+
+This [video](https://www.crowdcast.io/e/defi-discussions/18) will walk through the steps of this tutorial, starting around minute 11:00.
+
+## Prerequisites
+
+1. Install homebrew via website `brew.sh`
```

Comment:
this could be a link to a tutorial to install this perhaps?  
```suggestion
1. Install homebrew via Homebrew. Tutorial can be found [here](https://changelog.com/posts/install-node-js-with-homebrew-on-os-x).
```